Resources/Private/JavaScript/store.svelte.js

```
@@ -10,6 +10,54 @@
+        case '<':
+            return parseInt(point[field]) < parseInt(value);
+        case '>=':
+            return parseInt(point[field]) >= parseInt(value);
+        case '<=':
+            return parseInt(point[field]) <= parseInt(value);
+        case 'IN':
+            return value.split(',').includes(point[field]);
+        case '!IN':
+            return !value.split(',').includes(point[field]);
+        case '-':
+            const [min, max] = value.split('-');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)